modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements_controller.rb

```diff
@@ -79,7 +79,8 @@ def new_notice_of_disagreement
     @notice_of_disagreement = AppealsApi::NoticeOfDisagreement.new(
       auth_headers: headers,
       form_data: @json_body,
-      source: headers['X-Consumer-Username']
+      source: headers['X-Consumer-Username'],
+      board_review_option: @json_body['data']['attributes']['boardReviewOption']
```

Comment:
It's unfortunate that we need this, but there's no defensive measure here in case of `nil`. What happens if `['data']['attributes']['boardRevviewOption']` is nil?

Reply:
right right right blah!

Reply:
Well, wait, if board review is nil...schema error...it's required and will get caught in create when it goes to save the object.

Reply:
It validates the JSON schema before `new_notice_of_disagreement`, so it would never get to this point and be nil.